SohamGaikwad/Beginner/ContactApp/app/src/main/java/com/example/contactapp/ContactAdapter.java

```diff
@@ -0,0 +1,91 @@
+package com.example.contactapp;
+
+import android.content.Context;
+import android.content.Intent;
+import android.graphics.Bitmap;
+import android.graphics.BitmapFactory;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.ImageView;
+import android.widget.TextView;
+
+import androidx.annotation.NonNull;
+import androidx.recyclerview.widget.RecyclerView;
+
+
+import java.io.ByteArrayOutputStream;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.util.ArrayList;
+
+public class ContactAdapter extends RecyclerView.Adapter<ContactAdapter.ViewHolder> {
```

Comment:
Remove unused imports, functions, proper scope definition missing. 
Inner class 'ViewHolder' may be 'static'

Reply:
Fixed in ad34fcde338dffceb4d0ffa797d1c1bfc738641e.